opentelekomcloud/provider_test.go

```diff
@@ -35,6 +35,7 @@ var (
 	OS_BMS_FLAVOR_NAME        = os.Getenv("OS_BMS_FLAVOR_NAME")
 	OS_NIC_ID                 = os.Getenv("OS_NIC_ID")
 	OS_TO_TENANT_ID           = os.Getenv("OS_TO_TENANT_ID")
+	OS_TENANT_NAME            = os.Getenv("OS_TENANT_NAME")
```

Comment:
OS_TENANT_NAME may be fetch from environment of "OS_PROJECT_NAME". see  https://github.com/terraform-providers/terraform-provider-opentelekomcloud/blob/master/opentelekomcloud/provider.go#L71-L73

Reply:
done